src/styles/card.scss

```diff
@@ -0,0 +1,100 @@
+@import './var';
+@import './main';
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  height: 400px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+
+  &__top {
+    display: flex;
+    justify-content: center;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__top-img {
+    width: 100%;
+  }
+
+  &__bottom {
+    margin-top: 40%;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $mainAccent;
+  }
+
+  &__bottom-code {
+    font-size: 10px;
+    font-weight: 400;
+    margin-top: 4px;
+    line-height: 14px;
+    color: $secondary;
+  }
+
+  &-stars {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 16px;
+  }
+
+  &-text {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $mainAccent;
+  }
+
+  &-cost {
```

Comment:
The class name `&-cost` should be `&__bottom-cost` to match the HTML file's class name `card__bottom-cost`.